libse/SubtitleFormats/AdvancedSubStationAlpha.cs

```diff
@@ -420,15 +420,16 @@ public static List<string> GetStylesFromHeader(string headerLines)
         public static string FormatText(Paragraph p)
         {
             string text = p.Text.Replace(Environment.NewLine, "\\N");
-            text = text.Replace("<i>", @"{\i1}");
-            text = text.Replace("</i>", @"{\i0}");
-            text = text.Replace("</i>", @"{\i}");
-            text = text.Replace("<u>", @"{\u1}");
-            text = text.Replace("</u>", @"{\u0}");
-            text = text.Replace("</u>", @"{\u}");

```

Comment:
Removed.
After https://github.com/SubtitleEdit/subtitleedit/pull/2370/files#diff-f45841b5603f3cf09ce998e01cd59718L427 there will be no `</u>`